test/async_scope/test_stop.cpp

```diff
@@ -41,12 +41,12 @@ TEST_CASE("TODO: cancelling the associated stop_source will cancel the async_sco
   // REQUIRE(P2519::__scope::empty(scope));
   impulse_scheduler sch;
   scope.spawn(ex::on(sch, ex::just()));
+
+  sch.start_next();
+
   // TODO: the scope needs to be empty
   // REQUIRE(P2519::__scope::empty(scope));
   // REQUIRE_FALSE(P2519::__scope::empty(scope));
```

Comment:
?

Reply:
@lucteo wrote these and disabled them - I would like to have him take a look at these tests